docs/LSPApi.md

```diff
@@ -59,6 +60,10 @@ public class MyLanguageServerFactory implements LanguageServerFactory {
 | URI getFileUri(VirtualFile file)                             | Returns the file Uri from the given virtual file and null otherwise (to use default `FileUriSupport.DEFAULT.getFileUri`).                                                                                                                                                                                                                                                                      | `null`            |
 | VirtualFile findFileByUri(String fileUri)                    | Returns the virtual file found by the given file Uri and null otherwise (to use default `FileUriSupport.DEFAULT.findFileByUri`).                                                                                                                                                                                                                                                               | `null`            |
 | boolean isCaseSensitive(PsiFile file)                        | Returns `true` if the language grammar for the given file is case-sensitive and `false` otherwise.                                                                                                                                                                                                                                                                                             | `false`           | 
+| String getLineCommentPrefix(PsiFile file)                    | Returns the language grammar line comment prefix for the file.                                                                                                                                                                                                                                                                                                                                 |                   | 

```

Comment:
Added a top-level client configuration property for the language grammar's comment prefixes/suffixes and statement terminator characters since those aren't available (reliably) from LSP, TextMate, abstract file type, etc.